Prolangle/Components/LanguageMetadatum.razor

```diff
@@ -18,16 +19,24 @@
 
 	any = true;
 
-	var className = Overlap.HasFlag(flag) ? "overlap" : "";
+	var cssClassBuilder = new CssBuilder();
+
+	if (Overlap.HasFlag(flag))
+		cssClassBuilder.AddClass("overlap");
 
 	var desc = MetadataProvider.ResolveDescription(flag);
 	var moreInfoUrl = MetadataProvider.ResolveMoreInfoUrl(flag);
 
-	var style = string.IsNullOrWhiteSpace(desc) ? "" : "text-decoration: underline; text-decoration-style: dotted; text-underline-offset: 2px; text-decoration-thickness: 1px; cursor: help;";
+	var style = "";
+	if (!string.IsNullOrWhiteSpace(desc))
+	{
+		cssClassBuilder.AddClass("popover-available-hint");
+		style = "cursor: help";
```

Comment:
Maybe we should move this part of the style to the class as well

Reply:
I thought it would be confusing if the explanation gets a help cursor but doesn't actually behave that way?

Reply:
true